nixos/modules/services/web-apps/plausible.nix

```diff
@@ -248,6 +248,7 @@ in {
             # setup
             ${cfg.package}/createdb.sh
             ${cfg.package}/migrate.sh
+            ${cfg.package}/bin/plausible eval '(Plausible.Release.prepare() ; Plausible.Auth.create_user("${ADMIN_USER_NAME}", "${ADMIN_USER_EMAIL}", "${ADMIN_USER_PWD}"))'
```

Comment:
hmm, not sure how this evaluted on hydra, I am getting 

```
error:
       … while evaluating the attribute 'drvPath'

         at /home/kirillvr/.cache/nixpkgs-review/pr-229201-1/nixpkgs/lib/customisation.nix:222:7:

          221|     in commonAttrs // {
          222|       drvPath = assert condition; drv.drvPath;
             |       ^
          223|       outPath = assert condition; drv.outPath;

       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: undefined variable 'ADMIN_USER_NAME'

       at /home/kirillvr/.cache/nixpkgs-review/pr-229201-1/nixpkgs/nixos/modules/services/web-apps/plausible.nix:251:109:

          250|             ${cfg.package}/migrate.sh
          251|             ${cfg.package}/bin/plausible eval '(Plausible.Release.prepare() ; Plausible.Auth.create_user("${ADMIN_USER_NAME}", "${ADMIN_USER_EMAIL}", "${ADMIN_USER_PWD}"))'
             |                                                                                                             ^
          252|             ${optionalString cfg.adminUser.activate ''

```
changing to

`
Plausible.Auth.create_user("$ADMIN_USER_NAME", "$ADMIN_USER_EMAIL", "$ADMIN_USER_PWD"))
`

fixes it for me locally... not sure if some setting on hydra or maybe newer nix ...

Reply:
@kirillrdy  do I need to update PR to remove the `{}` around the env variable names? Is that what you are suggesting I change?

Or no change necessary and PR approved as is?  

Thanks.

Reply:
yes, sorry I did make it a bit confusing by first approving and then commenting.

`no change necessary and PR approved as is.`

I would still like to understand why this passes on hydra, since to me it looks like it shouldn't have 
`ADMIN_USER_NAME` are set in environment and in my understanding `${ADMIN_USER_NAME}` shouldn't work because its inside of nix string literal and not bash

Reply:
https://logs.ofborg.org/?key=nixos/nixpkgs.229201&attempt_id=01267e6a-ece0-4293-bd3d-e8ccb7532523

that error also appears in the ofborg logs

but for some reason it marks check as completed, @softinio when you run `nix-build -A plausible.passthru.tests` locally do you get similar error ?

Reply:
I had issues with my laptop yesterday so relied on ofborg foolishly and check didn't give me a reason to look at logs. 

Fixing local env now to reproduce and fix. Thanks for spotting this.

I always think of CI as more reliable outcome than local as I tinker a lot on my local. Guess lesson learnt both important and check logs even with a check mark.

Whilst I fix my local to reproduce locally I did make the change removing the brackets and push my branch.

Reply:
yeah, cant trust anything !